hs_core/templates/pages/genericresource.html

```diff
@@ -598,18 +598,17 @@ <h4 class="modal-title">Confirm files deletion</h4>
                                                 </a>
                                             {% endif %}
 
-                                            {% for b in cm.bags.all %}
```

Comment:
Really just curious why we had a list of bags.  Were there plans at one time to create mutliple bags?